app/views/katello/api/v2/content_facet/base.json.rabl

```diff
@@ -1,6 +1,12 @@
-attributes :content_view_id, :content_view_name
-attributes :lifecycle_environment_id, :lifecycle_environment_name
-attributes :uuid
+child :content_view => :content_view do
+  attributes :id, :name
+end
+
+child :lifecycle_environment => :lifecycle_environment do
+  attributes :id, :name
+end
```

Comment:
I look at it this way, what we feel is the best format is relevant; but we have told users that the API can be interacted with in a particular way and we owe it to them to abide by that contract (http://theforeman.org/manuals/1.10/index.html#5.1.7HandlingAssociations). If having both provides tangible benefits without overhead, then go for it. If it's just stupid and annoying, then we should start curating this information to inform the next API version.